packages/ui/src/Providers/IframeModalProvider/IframeModal/index.tsx

```diff
@@ -52,7 +52,8 @@ const IframeModal = ({ className, title = '', href = '', onClose }: ModalProps)
           <iframe
             title={title}
             src={href}
-            sandbox="allow-scripts"
+            // Applies all restrictions
+            sandbox=""
```

Comment:
will this disable SPA?

Reply:
It will. Should we only accept static HTTP to display the terms?

Reply:
Agree with static HTTP